cob_bringup/robots/cob4-2.xml

```diff
@@ -266,6 +266,14 @@
 			<arg name="robot" value="$(arg robot)" />
 		</include>
 
+		<include file="$(find cob_bringup)/drivers/usb_camera_node.launch" />
+
+                <include file="$(find cob_bringup)/drivers/image_flip.launch">
+                        <arg name="robot" value="$(arg robot)"/>
+                        <arg name="pkg_hardware_config" value="$(arg pkg_hardware_config)"/>
```

Comment:
the launch argument pkg_hardware_config is not set within cob4-2.xml. It has a default value within image_flip.launch which is good for all default cob setups.

So please remove this line.